Update the default value of `pd_enable_follower_handle_region` to ON (#20477)
pingcap/docs: system-variables.md

- Persists to cluster: Yes
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
- Type: Boolean

> - Default value: `ON` (Before v9.0.0, the default value is `OFF`)

Reviewer: ```suggestion
- Default value: `ON`. Before v9.0.0, the default value is `OFF`.
```

> - This variable controls whether to enable the [Active PD Follower](/tune-region-performance.md#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature.

Reviewer: ```suggestion
- This variable controls whether to enable the [Active PD Follower](/tune-region-performance.md#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature, which currently only applicable to requests for Region information.
```

> - This variable controls whether to enable the [Active PD Follower](/tune-region-performance.md#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature, which currently only applicable to requests for Region information.

Author: ```suggestion
- This variable controls whether to enable the [Active PD Follower](https://docs.pingcap.com/tidb/dev/tune-region-performance/#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature, which currently only applicable to requests for Region information.
```

    - When it is `OFF`, TiDB only obtains Region information from the PD leader.

>     - When it is `ON`, TiDB evenly distributes Region information requests to all PD servers, so PD followers can also handle Region requests, reducing the CPU pressure on the PD leader. Starting from v9.0.0, Region information requests from TiDB Lightning and BR are also evenly sent to all PD nodes when the value is `ON`.

Author: ```suggestion
    - When it is `ON`, TiDB evenly distributes Region information requests to all PD servers, so PD followers can also handle Region requests, reducing the CPU pressure on the PD leader. Starting from v9.0.0, Region information requests from TiDB Lightning are also evenly sent to all PD nodes when the value is `ON`.
```

Author: https://github.com/pingcap/docs-cn/pull/20000/files#r2011627377
